### ISSUE TITLE
Fire Emblem Engage: Game does not load with DLC installed

### ISSUE THREAD

**Issue by replicaJunction** [yuzu-emu/yuzu#11595]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Mainline 1568

### Description of Issue

Fire Emblem: Engage loads normally when no add-on content is installed. It also loads with the update 2.0.0 file. However, as soon as I install either DLC 1 or 2, the game fails to load and gets stuck on a black screen. The "launching" screen disappears, but the game's title screen does not appear, and the FPS indicator drops to zero.

### Expected Behavior

The game should load the DLC content normally.

### Reproduction Steps

1. Dump Fire Emblem Engage from cartridge
2. Dump Update 2.0.0, DLC 1, and DLC 2 from NAND
    - Fire Emblem Engage v327680 (0100A6301214E800) (UPD).nsp
    - Fire Emblem Engage v0 (0100A6301214F001) (DLC).nsp
    - Fire Emblem Engage v0 (0100A6301214F002) (DLC).nsp
3. Transfer all files to the computer
4. Start Yuzu and launch Fire Emblem Engage without installing any update files
    - Game loads as expected
5. Install the UPD file to NAND and load the game
    - Game loads as expected
6. Install the DLC files to NAND and load the game
    - Game does not load

### Log File

Attached

### System Configuration

CPU: AMD Ryzen 7 3700X
GPU/Driver: NVIDIA GeForce RTX 3060 / driver version 537.42 (2023-09-21)
RAM: 32GB
OS: Windows 11 Version 22H2 (Build 22621.2283)

**Comment by replicaJunction** [comment]
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/12719820/yuzu_log.txt)

**Comment by liamwhite** [comment]
Cannot reproduce. I noticed that you have many dumped files with invalid (empty) tickets -- you might want to check your dump options to see if there is anything unusual.

**Comment by replicaJunction** [comment]
Interesting. Thanks for looking at it.

I had a bad dump of a different game in the same subfolder (Three Hopes). I dumped that again and resolved the errors in the log that said `Crypto <Warning> core\crypto\key_manager.cpp:Read:222: Attempted to read ticket file with invalid size 0.` However, that didn't fix the issue here with Engage.

I don't see any settings out of the ordinary in NXDumpTool. I've verified that I'm using the settings from the [documentation](https://yuzu-emu.org/help/quickstart/#dumping-games), and I re-dumped the update and the DLC files to be sure. Same behavior.

Here is an updated log file with the ticket warnings resolved:

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/12721391/yuzu_log.txt)

I respect that you can't reproduce it. What else can I try to provide more helpful information?

**Comment by replicaJunction** [comment]
Okay, I've fixed my own issue.

1. Started the game on my Switch (with Internet access and no CFW loaded). I'd downloaded the DLC but hadn't started the game again since.
2. Re-dumped the two DLC files.
3. Re-dumped my title keys. Not sure why, but the title.keys I had in Yuzu was 1KB smaller than the new one I dumped.

After doing these three, the game now boots reliably with the DLC installed again. Thanks again for looking at the issue.
